### PR TITLE
Use CodeMirror for fixture editor

### PR DESCRIPTION
**Pull request by bogdan-stefan-hs**
## Need

``` gherkin
As a user
I want a powerful and feature-rich editor
So that I can edit fixtures fast and easy.
```
## Deliverables
- [x] A `CodeMirror` instance with code editor features, as opposed to a plain `textarea`. This editor will have the following features: 
  - Code folding
  - Syntax highlighting
  - Smart indentation
  - Line numbers
## Solution
- Use [`react-codemirror`](https://www.npmjs.com/package/react-codemirror) for a CodeMirror React component and adapt it for JSON syntax.

### CODE REVIEW

**Comment by bogdan-stefan-hs** [comment]
@valentin-radulescu-hs @andrei-picus-hs Please review code.

image

File: src/components/component-playground.jsx [hunk 1]
```diff
@@ -254,13 +293,7 @@ module.exports = React.createClass({
 
     return <div key="fixture-editor-outer"
                 className={style['fixture-editor-outer']}>
-      <textarea ref="editor"
-                className={editorClasses}
```
Comment (valentin-radulescu-hs) [inline src/components/component-playground.jsx:258]: You can remove the `editorClasses` variable as well.

**Comment by valentin-radulescu-hs** [comment]
Finished review, LGTM

**Pull request merged**